sessionctx/variable/session.go

```diff
@@ -387,6 +390,7 @@ func NewSessionVars() *SessionVars {
 		L2CacheSize:               cpuid.CPU.Cache.L2,
 		CommandValue:              uint32(mysql.ComSleep),
 		SlowQueryFile:             config.GetGlobalConfig().Log.SlowQueryFile,
+		EnableFastAnalyze:         false,
```

Comment:
No need for it? The default value for `bool` is false.

Reply:
Removed. And btw, `builtinGlobalVariable` has also been added.